Assets/SC_Attackforenemyclassobjects.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SC_Attackforenemyclassobjects : MonoBehaviour
```

Comment:
The name of the class is unintuitive and incomprehensible. It would be better to name the class using PascalCase.
```suggestion
public class EnemyAttackScript: MonoBehaviour
```